nightly-style-policy.md

```diff
@@ -0,0 +1,26 @@
+Initial PR(s) implementing new syntax filed against rust-lang/rust should
+generally leave the rustfmt subtree untouched. In cases that that need to
+modify rustfmt (for example, to fix compiler errors that come from adding
+new variants to AST representation), they should modify rustfmt in such a
+way to keep existing formatting verbatim.
+
+Rustfmt is allowed to implement nightly-only formatting behavior without that
+syntax being specified by the style guide. Initial PR implementors are 
+encouraged but not required to open a PR against rustfmt suggesting an initial
```

Comment:
A few small changes suggested all as one. Don't feel strongly about any, feel free to take/leave any/all
```suggestion
syntax being specified by the style guide. The initial authors of PR implementing new features in rust-lang/rust are 
encouraged, but not required, to open a PR against [rustfmt](https://github.com/rust-lang/rustfmt) suggesting an initial
```